configure.ac

```diff
@@ -274,12 +274,25 @@ AC_ARG_ENABLE([suid],
 
 AS_IF([test "x$enable_suid" != "xno"], [
     AC_MSG_RESULT([yes])
-    BUILD_SUID="action-suid create-suid copy-suid expand-suid export-suid import-suid mount-suid"
+    BUILD_SUID="action-suid copy-suid expand-suid export-suid import-suid mount-suid"
```

Comment:
That happens on Line 291: it `$BUILD_SUID` is non-empty, we add `create-suid` to the list:
```
AS_IF([test -n "$BUILD_SUID"], [BUILD_SUID="$BUILD_SUID create-suid"], [])
```